### ISSUE TITLE
PeoplePicker A11y Pass

### ISSUE THREAD

**Issue by micahgodbolt** [OfficeDev/office-ui-fabric-react#6259]
Tracking all of the issues brought up with the 08/02 accessibility pass on People Picker

Details on individual bugs can be found on [SharePoint](https://microsoft-my.sharepoint.com/:f:/p/v-amaver/EnpHiS4SJT9AmpvoASiG4YkB0tjIkeReAXeheqfjbv6FYw?e=rT8roS)

- [x] Bug 1 : [Accessibility][Programmatic access]: ARIA attributes must conform to valid values for 'People picker control'.
- [x] Bug 2 : [Accessibility][Programmatic access]: Certain ARIA roles must contain particular children.
- [ ] Bug 3 : [Accessibility][Functional]: ~~User should not be allowed to add invalid contacts in 'People picker control'.~~ * Wont' fix
- [x] Bug 4* : [Accessibility][Keyboard Navigation]: When user navigates ahead from the 'People picker combo box' using tab key first contact in 'Suggested contact' list gets selected by itself. 
- [ ] Bug 5 : [Accessibility][Keyboard Navigation]: Arrow key focus moves out of the 'Suggested Contacts' list window in forward and backward navigation.
- [ ] Bug 6 : [Accessibility][Programmatic access]: Role of the 'Process selection contacts' should be menu item.
- [x] Bug 7: [Accessibility][Programmatic access]: Narrator does not give any feedback when user selects or deletes any people contact from 'People picker combo box'.
- [x] Bug 8* : [Accessibility][Screen reader]: Narrator does not convey 'People picker' label when first time user navigates on 'people picker combo box'.
- [x] Bug 9* : [Accessibility][Screen reader]: When user searches any contact 'Suggested contact' list gets updated about which narrator does not convey anything.  
- [x] Bug 10 : [Accessibility][Keyboard navigation]: With Delay On, When user navigates on 'Suggested contact' using down arrow key focus does not go on first list item.
- [ ] Bug 11* : [Accessibility][Programmatic access]: Contact 'Remove button' does not have name.
- [x] Bug 12 : [Accessibility][Keyboard navigation]: After Removing any contact from list, Keyboard focus jumps to the 'People picker' control.
- [ ] Bug 13 : [Accessibility][Keyboard navigation]: User is not able to delete contacts in 'People picker combo box' using 'Enter' key. [Duplicated by #5287]
- [ ] Bug 14 : [Accessibility][Form and validations]: 'Click to Add a person' label is not associated with the first contact list item.
- [ ] Bug 15* : [Accessibility][Keyboard navigation]: After selecting last contact from 'Process selection contact' list, keyboard focus gets trapped somewhere and become invisible.
- [ ] Bug 16 : [Accessibility][Keyboard navigation]: 'Remove' button does not appear in 'Suggested contact list' window on keyboard focus and is also not accessible. [Duplicated by #4816]
- [ ] Bug 17 : [Accessibility][Functional]: Incorrect label appears when no contacts available to add in 'People picker' combo  box.
- [ ] Bug 18 : [Accessibility][Supporting the Platform]: 'Suggested contacts' does not adopt high contrast 'Selected text' setting.
- [ ] Bug 19 : [Accessibility][Keyboard navigation]: 'Selected Contacts' should remove on pressing one single enter key.  
- [ ] Bug 20 : [Accessibility][Usable]: 'Auto complete values' gets saved in 'People picker combo box' even if user does not select any value.
- [ ] Bug 21 : [Accessibility][Usable]: Multiple focuses provided on single contact list item in scan mode.
- [x] Bug 22 : [Accessibility][Usable]: In Scan mode 'Suggested contacts window is not accessible using arrow key.
- [x] Bug 23 : [Accessibility][Usable]: Narrator focus doesn't land on the next user while removing the current user in People picker.

**Comment by micahgodbolt** [comment]
Consider this a list of items to triage. Additional issues can be created and linked back here if necessary.

**Comment by dzearing** [comment]
@joschect ping.

**Comment by dzearing** [comment]
@joschect ping.

**Comment by joschect** [comment]
Broken out into smaller issues.
